### ISSUE TITLE
About interactiveness AP

### ISSUE THREAD

**Issue by mrwu-mac** [enlighten0707/Body-Part-Map-for-Interactiveness#7]
Thank you for your work.

I have a little issue about computing interactiveness AP. I found that you compute it only based on box score instead of considering the binary score in `binary_evaluation.py`. So I confuse about this.

**Comment by mrwu-mac** [comment]
Oh, I may have been misled by the function `Generate_HICO_detection`.

**Comment by yuzheyao22** [comment]
One thing that confused me is when printing the "vs"(verb_scores) in hoi.py, logits for all the 117 verbs are the same. It means that the model does not predict the verb, right?
